### ISSUE TITLE
aarlo.camera.request_video_to_file service only works if there are cloud recordings

### ISSUE THREAD

**Issue by bennyb0i** [twrecked/hass-aarlo#516]
Hey there; I've had hass-aarlo up and running for some time and it's working great. Even put together a fairly robust Home Assistant blueprint for automating DOODS image processing and mobile notifications that I'm considering to make public. Anyway, I recently cut the cord from my pesky Arlo subscription and, with that, my Arlo Doorbell also gave up its cloud recordings. Was thinking this wouldn't be a problem since with hass-aarlo you can just invoke the service `aarlo.camera_request_video_to_file` and (according to the description at least) it would take a video and write it to a local file. However, I've noticed that invoking this service only takes the most recent cloud recorded video and writes it to a local file instead. So, sadly, if there's no cloud recordings present, invoking the service just coughs up an "Unknown error". Is this service working as intended? If so, is there another way to have the camera invoke a 'live' recording and save it to a local file, perhaps using the stream function or some such?

**Comment by twrecked** [comment]
That service just downloads the last recorded video. I think the `camera.record` service is the one you want.

Let me know if it works.

**Comment by bennyb0i** [comment]
Yep, I actually discovered earlier that 'camera.record' does indeed work, thanks! Though, sadly it doesn't look like it's going to be a viable solution since it takes at least several seconds to connect to the camera stream, and by that time a bulk of the event is often missed. In the end I just ponied up again for the Arlo Smart subscription for the doorbell camera only. Still saving money versus having Arlo Smart on all my other cameras as they're Arlo Pro/Pro2 (i,e,, free 7-day cloud storage). A shame Arlo decided not to give the same 7-day cloud storage on its newer devices since then.
